src/EditIdentitiesForm.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <form @submit.prevent>
+    <draggable
+      v-model="identities"
+      :options="draggableOptions"
+      @end="handleReorderIdentities"
+    >
+      <Identity
+        v-for="identity in identities"
+        :key="identity.id"
+        :identity="identity"
+        :disabled="identity.disabled"
+        @updateIdentity="handleUpdateIdentity"
```

Comment:
I actually don't know why I decided to bubble things up, at this point; maybe to minimize the repetition of the API glue I had before adding in axios. Now it's trivial to handle save, at least, inside the Identity component. Delete will require the parent to splice the element out, though, so that should bubble.

That said, I do like your point about getting rid of `handle`. DRF does something similar to what I'm doing here, albeit along a different axis, where it has:

 - `.get`
 - `.post`
 - `.put`
 - `.delete`

methods that just handle the appropriate HTTP verbs, and then:

 - `.list`
 - `.create`
 - `.retrieve`
 - `.update`
 - `.destroy`

methods that actually do the work.

The dimension along which these are differentiated, though, is worth considering in the naming. The `handle` versions call out to an API helper, and then do DOM modification work to keep the representation in sync. I'll consider it.